.github/workflows/nestbuildmatrix.yml

```diff
@@ -666,6 +659,7 @@ jobs:
               -Dwith-warning=${{ contains(matrix.use, 'warning') && 'ON' || 'OFF' }} \
               -Dwith-boost=${{ contains(matrix.use, 'boost') && 'ON' || 'OFF' }} \
               -Dwith-openmp=${{ contains(matrix.use, 'openmp') && 'ON' || 'OFF' }} \
+              ${{ contains(matrix.use, 'openmp') && '-DOpenMP_ROOT=`brew --prefix libomp`' }} \
               -Dwith-mpi=${{ contains(matrix.use, 'mpi') && 'ON' || 'OFF' }} \
```

Comment:
Check the logs for the MPI config in the `cmake` step. There seems to be some mixup between `brew` installed and system installed MPI versions:
```
 Use MPI             : Yes (MPI: /usr/local/bin/mpicxx)
    Includes        : /usr/local/Cellar/open-mpi/4.1.5/include
    Libraries       : /usr/local/Cellar/open-mpi/4.1.5/lib/libmpi.dylib
    Link Flags      : -L/usr/local/Cellar/libevent/2.1.12/lib
    Launcher        : /usr/local/bin/mpiexec -n <np> <prog> <args>
```

Reply:
Welcome to the wondrous world of brew. The binaries in `/usr/local/bin` are symlinks to the actual files in `/usr/local/Cellar/open-mpi/...`, so this is consistent.